backend/main.go

```diff
@@ -11,6 +11,11 @@ import (
 	"encoding/base32"
 	"flag"
 	"fmt"
+	"net/http"
```

Comment:
for some reason vsc always changes these to have this format, I mean, looks cleaner but still annoying it always adds it...